packages/pwa-kit-mcp/src/tools/create-new-page-tool.test.js

```diff
@@ -117,10 +117,17 @@ describe('CreateNewPageTool', () => {
         jest.spyOn(createNewPageTool, 'generatePageContent').mockResolvedValue(
             `const productId = '25592300M';\nexport default function Page() { return <div>{productId}</div>; }`
         )
+        const internalPaths = {
+            nodeModulesPath: '/mock/node_modules',
```

Comment:
I assume these variable names are internal and don't need to be updated to use snake care (with underscores) also. Is this right?

Reply:
yes thats correct, this is a test file